docs/tutorials/go/audiobook/index.md

```diff
@@ -0,0 +1,1026 @@
+```
+
+Once running, connect to the [Temporal Web UI](http://localhost:8080/) and verify that the server is working.
+
+### Instantiate Your Bearer Token
+
+Create an environment variable called OPEN_AI_BEARER_TOKEN to configure your OpenAI credentials.
+If you set this value using a shell script, make sure to `source` the script so the variable carries over past the script execution.
+The environment variable must be set in the same shell where you'll run your application.
+
+### Run the Worker application
+
+1. Update your dependencies:
+
+   ```sh
+   go mod tidy
+   ```
+
+1. Build the Worker app:
+
+   ```sh
+   make
+   ```
+
+2. Start the app running:
+
+   ```sh
+   make run
+   ```
+
+If the Worker can't fetch a bearer token from the shell environment, it will fail loudly at launch.
+This early check prevents you from running jobs and waiting to find out that you forgot to set the bearer token until you're well into the Workflow process.
+
+Now that the Worker is running, you can submit jobs for text-to-speech processing.
+
+## Submit narration jobs
+
+For this tutorial, use the Temporal CLI tool to build audio from text files.
+Open a new terminal window.
+You'll use the Workflow `execute` subcommand to watch the execution in real time from your command line:
+
+```sh
+temporal workflow execute \
+    --type TTSWorkflow \
+    --task-queue tts-task-queue \
+    --input '"/path/to/your/text-file.txt"' \
+    --workflow-id "your-workflow-id"
+```
+
+* **type**: The name of this text-to-speech Workflow is `TTSWorkflow`.
+* **task-queue**: This Worker polls the "tts-task-queue" Task Queue.
+* **input**: Pass a quoted JSON string with a `/path/to/your/input/text-file`.
+* **workflow-id**: Set a descriptive name for your Workflow Id.
+  This makes it easier to track your Workflow Execution in the Web UI.
+  The identifier you set doesn't affect the input text file or the output audio file names.
+
+For example, you might run:
+
+```sh
+temporal workflow execute \
+    --type TTSWorkflow \
+    --task-queue tts-task-queue \
+    --input '"~/Desktop/chapter-1.txt"' \
+    --workflow-id "chapter-1-tts"
+```
+
+### Find your output file
+
+Your output is collected in a system-provided temporary file.
+After, your generated MP3 audio is moved into the same folder as your input text file.
+It uses the same name replacing the `txt` extension with `mp3`.
+If an output file already exists, the project versions it to prevent name collisions.
+
+The `TTSWorkflow` returns a string, the `/path/to/your/output/audio-file`.
+Check the Web UI Input and Results section after the Workflow completes.
+The results path is also displayed as part of the CLI's `workflow execute` command output and in the Worker logs.
+
+:::tip Cautions and notes
+
+- Do not modify your input or output files while the Workflow is running.
+- The Workflow fails if you don't pass a valid text file named with a `txt` extension.
+
+:::
+
+### Check your progress
```

Comment:
This part, and the above CLI submission step, are especially great -- would love to see them made more prominent, I think simplifying other parts of this tutorial will achieve that.